src/content/development/release-process/_index.en.md

```diff
@@ -36,19 +36,20 @@ Version numbers are required to be formatted following the schema norms where th
 
 ### Creating Releases
 
-The following sections are an ordered series of steps to create a Submariner relese.
+The following sections are an ordered series of steps to create a Submariner release.
 
-The release process is mostly automated and uses a YAML file created in the releases repository that describes the release.
+The release process is mostly automated and uses a YAML file created in the
+[releases repository](https://github.com/submariner-io/releases) that describes the release.
 This file is updated for each step in the release process.
 
 Once the changes for a step are reviewed and merged, a CI job will run to create the release(s) for the step and create the
 required pull requests in preparation for the next step to be reviewed and merged. Once all these pull requests have been
 merged, you can continue onto the next step.
 
-For most projects, after a release is created, another job will be initiated to build release artifacts and publish to Quay.
+For most projects, after a release is created, another job will be initiated to build release artifacts and publish it to Quay.
```

Comment:
```suggestion
For most projects, after a release is created, another job will be initiated to build release artifacts and publish them to Quay.
```

I also think it's okay as-is, "publish to Quay".

Reply:
Reverting to what it was originally.